ci/default.nix

```diff
@@ -75,6 +75,30 @@ let
           includes = [ "*" ];
           priority = 1;
         };
+
+        # TODO: Upstream this into treefmt eventually?
+        settings.formatter.markdown-code-runner = {
+          # Remove shell script once this is solved:
+          #   https://github.com/drupol/markdown-code-runner/issues/6
+          # Remove sed step once this is solved:
+          #   https://github.com/drupol/markdown-code-runner/issues/7
+          command =
+            let
+              config = pkgs.writeText "markdown-code-runner-config" ''
+                [presets.nixfmt]
+                language = "nix"
+                command = ["nixfmt"]
+              '';
```

Comment:
Can't we use something like https://noogle.dev/f/pkgs/writers/writeTOML instead ?